CMakeLists.txt

```diff
@@ -95,7 +95,9 @@ endif()
 
 # Tests
 
-include(CTest)
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
+    include(CTest)
+endif()
 
 if((CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME) AND BUILD_TESTING)
```

Comment:
Sorry for the late comment, but combining those to
```cmake
if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
    include(CTest)
    if (BUILD_TESTING)
        add_subdirectory(tests)
    endif()
endif()
```
might avoid a bit of duplication. Not an issue though, and it's merged already.